### ISSUE TITLE
Preloading proposals in user listings

### ISSUE THREAD

**Issue by januszm** [mynameisrufus/proposal#16]
I was wondering how to effectively preload roles for collections of users. Let's say I want to list a bunch of users and for each user I'd like to list her proposals. Currently I have a db query fired up for each user.

**Comment by mynameisrufus** [comment]
That's right, this is the tradeoff for 'unobtrusive'. Off the top of my head i'm not sure you could achieve `preload` without having to alter your schema and this would tightly couple the gem with the users application :(

**Comment by januszm** [comment]
That's ok. Keeping i "unobtrusive" is more important.
